lib/_stream_readable.js

```diff
@@ -458,6 +458,9 @@ Readable.prototype.read = function(n) {
       endReadable(this);
   }
 
+  // Drop data if _dumped flag exists
```

Comment:
Since the `_dumped` flag only exists in `http`, maybe mention that in the comment too so that it's clear that this bit of code is explicitly for `http` requests and what `_dumped` means.